File: Solutions/src/HotelWiring.cpp
/*
 IEEEXtreme 14.0 solutions
 Authors: team TLE - https://github.com/mhamedouadghiri - https://github.com/aaiit
 Verdict: 57.75 points (100%)
*/

#include <bits/stdc++.h>
using namespace std;

int main(int argc, char const *argv[])
{
    ios_base::sync_with_stdio(false);
    cin.tie(NULL);
    int q;
    cin >> q;
    while(q--)
    {
        int m, n, k;
        cin >> m >> n >> k;
        vector<int> f(m);
        for(int i = 0; i < m; i++)cin >> f[i];
        sort(f.begin(), f.end());
        long sum = 0;
        for(int i = 0; i < m; i++)
        {
            if(i < k)
            {
                sum += n - f[i];
            }
            else
            {
                sum += f[i];
            }
        }
        cout << sum << "\n";
    }
    return 0;
}

File: Solutions/src/Hotel Wiring.py
'''
 IEEEXtreme 14.0 solutions
 Authors: team seaholic - https://github.com/adnaneaabbar
 Verdict: 57.75 points (100%)
'''
for _ in range(int(input())):
    M = []
    l = list(map(int,input().split()))
    for _ in range(l[0]):
        M.append(int(input()))
    M.sort()
    for k in range(l[2]):

        M[k] = l[1] - M[k]

    print(sum(M))
